README.md

```diff
@@ -231,14 +231,19 @@ VELLO_STATIC_LOG="vello=trace" VELLO_STATIC_ARGS="--test-scenes" cargo apk run -
 
 ## Minimum supported Rust Version (MSRV)
 
-This version of Vello has been verified to compile with Rust 1.75 and later.
-
-Note that since then, one of Vello's dependencies might have released a new version with a higher requirement.
-Thus if you encounter a compilation issue due to a dependency, you should downgrade it via `cargo update -p package_name --precise 0.1.1` (with a real name and version).
+This version of Vello has been verified to compile with **Rust 1.75** and later.
 
 Future versions of Vello might increase the Rust version requirement.
 It will not be treated as a breaking change and as such can even happen with small patch releases.
 
+> [!TIP]
```

Comment:
See [#linebender > GitHub (markdown) alerts](https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/GitHub.20.28markdown.29.20alerts) on using these alerts. It has some unfortunate consequences

Reply:
I don't have strong feelings about this syntax, so I'll remove it from this PR so that discussion on the syntax can remain separated.